maven-plugin/src/main/java/org/openmrs/maven/plugins/BuildDistro.java

```diff
@@ -307,7 +307,7 @@ private void copyDockerfile(File targetDirectory, DistroProperties distroPropert
 				copyBuildDistroResource("Dockerfile-jre7", new File(targetDirectory, "Dockerfile"));
 			}
 		} else {
-			if (isPlatform2point5AndAbove(platformVersion)) {
+			if(isBetweenPlatform2point0and2point3(platformVersion)) {
```

Comment:
Have we tested this to verify that it works?

Reply:
Yes, I manually tested this.